packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-card.tsx

```diff
@@ -17,78 +17,42 @@ const JurisdictionCard = ({
                 onClick={disabled ? () => undefined : () => cardSelection(type_of_card)}
-                style={[1, 2, 3].includes(number_of_cards) ? { width: '32em' } : { width: '27.6em' }}
             >
-                {card_values.is_over_header_available && (
-                    <div className={classNames(`${card_classname}__over-header`)}>
-                        <Text as='p' color={'info-blue'} line_height='xxl' weight='bold'>
+                <div className={`${card_classname}__card-content-container`}>
+                    {card_values.is_over_header_available ? (
+                        <div className={`${card_classname}__card-content-over-header`}>
                             <Localize i18n_default_text={card_values.over_header} />
-                        </Text>
-                    </div>
-                )}
-                <div className={`${card_classname}__info-container`}>
+                        </div>
+                    ) : (
+                        <div className={`${card_classname}__card-content-over-header-blank`} />
+                    )}
                     <Text as='p' color={'prominent'} weight='bold' size='sm' className={`${card_classname}__h2-header`}>
                         <Localize i18n_default_text={card_values.header} />
                     </Text>
-                    {card_data.map((item, index) => (
-                        <div className={`${card_classname}__bullet-wrapper`} key={index}>
-                            <div>
-                                <Checkmark />
-                            </div>
-                            <Text as='p' size='xs' color={'prominent'}>
-                                <Localize i18n_default_text={item} />
-                            </Text>
-                            {/Straight-through processing/.test(item) && (
-                                <Popover
-                                    alignment='left'
-                                    className='cfd-compare-accounts-tooltip'
-                                    classNameBubble='cfd-compare-accounts-tooltip--msg'
-                                    icon='info'
-                                    disable_message_icon
-                                    is_bubble_hover_enabled
-                                    message={localize(
-                                        'Choosing this jurisdiction will give you a Financial STP account. Your trades will go directly to the market and have tighter spreads.'
-                                    )}
-                                    zIndex={9999}
-                                />
-                            )}
-                        </div>
-                    ))}
+                    <div className={`${card_classname}__card-section-container`}>
+                        {card_data.map((item, index) => (
+                            <>
+                                <JurisdictionCardSection key={index} cardSectionItem={item} />
```

Comment:
Instead of index, assign unique value from item as key

Reply:
@amina-deriv the array used is not going to change its element indices throughout the app lifecycle due to which changing this is not gonna make difference for react.. so shall we use the index instead of using/creating unique keys in the contents file?

Reply:
@shaheer-deriv the available jurisdictions  is not static and it always depends on the trading_platform_available_accounts API repsonse